manifests/00-crd-extension-console-cli.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: consoleclidownloads.console.openshift.io
+  annotations:
+    displayName: ConsoleCLIDownload
+    description: Extension for configuring openshift web console command line interface (CLI) downloads.
+spec:
+  scope: Cluster
+  group: console.openshift.io
+  versions:
+  - name: v1
+    served: true
+    storage: true
+  names:
+    plural: consoleclidownloads
+    singular: consoleclidownload
+    kind: ConsoleCLIDownload
+    listKind: ConsoleCLIDownloadList
+  # TODO: ensure path here is correct if we drop `spec`
+  additionalPrinterColumns:
+  - name: Display name
+    type: string
+    JSONPath: .spec.displayName
+  - name: Age
+    type: string
+    JSONPath: .metadata.creationTimestamp
+  - name: Description
+    type: string
+    JSONPath: .spec.description
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          type: object
+          description: Represents console CLI download spec
+          required:
+          - displayName
+          - description
+          - link
+          properties:
+            displayName:
+              type: string
+              description: Display name of the CLI download
+            description:
+              type: string
+              description: Description of the CLI download (can include markdown)
+            link:
+              type: object
+              description: Object that holds CLI download link details
+              required:
+              - href
+              properties:
+                text:
+                  type: string
+                  description: Text of the link
```

Comment:
`description: Text of the link.  If omitted, the text will default to "Download {displayName}".` ?
Not sure how helpful we want to make this.